ci-operator/step-registry/openstack/provision/bastionproxy/openstack-provision-bastionproxy-commands.sh

```diff
@@ -37,6 +39,10 @@ export OS_CLIENT_CONFIG_FILE="${SHARED_DIR}/clouds.yaml"
 WORK_DIR=${WORK_DIR:-$(mktemp -d -t shiftstack-ci-XXXXXXXXXX)}
 CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
 NET_ID=$(<"${SHARED_DIR}"/MACHINESSUBNET_NET_ID)
+SUBNET_ID=$(<"${SHARED_DIR}/"MACHINESSUBNET_SUBNET_RANGE)
```

Comment:
now that we have BASTION_NET_ID and NET_ID, I wonder if we could rename NET_ID, to MACHINES_NET_ID or something more obvious (not necessarily now).